apps/condo/domains/billing/schema/ValidateQRCodeService.test.js

```diff
@@ -165,26 +165,27 @@ describe('ValidateQRCodeService', () => {
             'BIC',
             'PayerAddress',
             'Sum',
-            'PaymPeriod',
-            'PersAcc',
+            'persAcc',
```

Comment:
Why did you make this parameter lowercase? And it seems that it's not consistent in the tests.

Reply:
We discovered that many real qr-codes does not meet the standart requirements for fields naming. So, we should work with all cases.

Reply:
btw, this case moved to the separated test